content/applications/general/email_communication/email_template.rst

```diff
@@ -0,0 +1,241 @@
+
+Because of this, setting a value in this field is useless as the value defined will be totally ignored. The default *reply-to* value is the default catchall email address to ensure a communication between your customer and your Odoo database. For more information about the way the catchall works, please check :ref:`how to manage inbound messages <inbound_messages>`.
+
+
+Transactional emails and corresponding URL for each company 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When using Odoo, multiple events trigger the sending of automated emails, these emails are known as transactional emails and sometimes contain links pointing to your Odoo database.
+
+By default, links generated by the database use the dynamic web.base.url key defined in the system parameters. More information about this parameter can be found `in our website documentation <https://www.odoo.com/documentation/latest/applications/websites/website/publish/domain_name.html#how-to-make-sure-that-all-my-urls-use-my-custom-domain>`_.
+
+If the website application isn't installed, the web.base.url key will always be the default parameter used to generate all the links.
+
+It’s important to know that this key can only have a single value, meaning that in a multi-website/company database environment, even if you have a specific domain name for each website, the links generated to share a document or within a transactional email might remain the same, whatever the website/company related to the sending of the email/document.
+
+This is not always the case as some Odoo applications have a link established in the database with the website application, meaning that in this case, if a specific domain is defined for the websites, the URL generated in the email template will use the domain defined on the corresponding website of the company.
+
+For example:
+A document shared using the documents application will always use the web.base.url key, as the document shared isn't associated with any particular website. Meaning that the URL will always be the same (the web.base.url key value) , whatever the company it's shared from, this is a known limitation!
+
+On the other hand, sales orders made by a customer on one of your Odoo e-commerce websites have a link established with the website from which the order was made. As a result, the e-mail sent for the sales orders uses the domain name defined for the corresponding website to generate the links.
+
+For more information about how to configure your domains, we invite you to check `our domain name documentation <https://www.odoo.com/documentation/latest/applications/websites/website/publish/domain_name.html>`_.
+
+Updating translations within email templates
+********************************************
+
+Email templates are automatically translated, changing the translations shouldn’t be necessary, however if for a specific reason you’d like to change some of the translations, this can be done.
```

Comment:
```suggestion
Email templates are automatically translated. Changing the translations shouldn’t be necessary. However, if for a specific reason you’d like to change some of the translations, this can be done.
```